smartlearner/interfaces/optimizer.py

```diff
@@ -79,3 +87,13 @@ def _apply_modifiers(self, list_modifiers, objects_to_modify):
             updates.update(modifier.updates)
 
         return updates
+
+    def save(self, path):
+        # TODO: should we save directions and params modifiers?
```

Comment:
I think the use case we had for param modifiers is clipping a param (e.g. needs to be > 0). I know you can do it with a directions modifier but you'll have to make sure it is the last modifier to be executed before the params update.

Regarding the question, we'll let the subclasses of `Optimizer` decide if they need to save them.